en/Plugins/Publish.md

```diff
@@ -1,43 +1,49 @@
 
-For more information about the Obsidian Publish service, what's included, how it works, and privacy go to the [[Obsidian Publish]] add-on service page.
+For more information about the Obsidian Publish™ service, what's included, how it works, and privacy, see the [[Obsidian Publish]] add-on service.
 
 ### Enable the plugin
 
-First, enable the Publish plugin. After that, you'll see the Publish icon appear in your [[Ribbon]], which looks like a paper plane.
+From within Obsidian > Settings > Plugin, enable the Publish plugin. After that, a Publish icon (which looks like a paper plane) will appear in the [[Ribbon]] pane.
 
 Once enabled, you can also set a hotkey for this option, or use the command palette to access it.
 
 ### Set up your site
 
-When purchasing Obsidian Publish, you'll be asked how many sites to purchase. You can then log in under Settings -> Account, so that the Publish plugin is aware of the number of remote sites you have.
+When purchasing Obsidian Publish, you'll be asked how many sites to purchase. Once you login under Settings > Account, the Publish plugin will display the number of publish sites you've purchased.
 
-If you have available slots, after opening Publish, you'll see an option to set up a site by entering a unique site ID. Your site will be available at `https://publish.obisidian.md/{site-id}`.
+If you have available sites, after opening Publish you'll see an option to set up a site by entering a unique site ID. After publishing, your site will be available at `https://publish.obisidian.md/{site-id}`.
 
 ### Site options
 
-After creating a site, you might want to give it a name, set a theme, and toggle the components (graph view, table of contents, etc.). You can do that by clicking on the gear icon near your site name, once you've created it.
+After creating a site, you have the option to give it a name, set a theme, and toggle various components (e.g. graph view, table of contents, etc.). You can do that by clicking on the gear icon near your site name at the top of the Publish plugin.
 
-### Upload files
+### Select files for publishing
 
-After setting up your site, you can now publish some notes!
+After setting up your site, you can now publish your notes!
 
-With a fresh, empty site, all the notes will appear under the "New" section, meaning they are new to your site.
+With a fresh, empty site, all your notes will appear under the "New" section, meaning they are new to your site.
 
-==Privacy note: at this stage, our Obsidian Publish server has no knowledge of these "New" notes. They are displayed as the Publish plugin compares what's in your vault with what's already on your published site.==
+==Privacy note: prior to your first publish, our Obsidian Publish server has no knowledge of these "New" notes. The Publish plugin compares what's in your vault with what's already on your published site and then displays any such unpublished notes.==
```

Comment:
Sorry for the "at this stage" comment. I think we can just delete this, as this note is not only true "prior to your first publish"; it's always true. Even after your first publish, the server still has zero knowledge of the "New" notes.

Reply:
Agreed. Although I do think if we take this out, we might want a privacy section that essentially says Obsidian Publish has no awareness (nor retention) of any notes you choose not to publish (or those you subsequently delete).